src/System.Numerics.Vectors/ref/System.Numerics.Vectors.csproj

```diff
@@ -4,8 +4,10 @@
   <PropertyGroup>
     <AssemblyVersion>4.1.1.0</AssemblyVersion>
     <OutputType>Library</OutputType>
-    <PackageTargetFramework>dotnet5.4</PackageTargetFramework>
-    <NuGetTargetMoniker>.NETPlatform,Version=v5.4</NuGetTargetMoniker>
+    <PackageTargetFramework>dotnet5.2</PackageTargetFramework>
```

Comment:
Let's not trust our thought experiment.  Try adding the validation and see if NuGet will use the lib for both compile and runtime.  If so, no need to duplicate the ref.